### ISSUE TITLE
local debug help

### ISSUE THREAD

**Issue by onesbyones** [hcoles/pitest#1173]
Expected: i hope create interceptor hook context java object and modify context java object

Actual: use debug run with pitest:mutationCoverage and set data.setVerbosity(VERBOSE), from the console log set logcal debug line number `72` on `MutationTestWorker.java` , why this application not run this line 

![image](https://user-images.githubusercontent.com/28530830/227156655-58e8bbad-f3af-43d3-a258-1846f17f32d4.png)

**Comment by hcoles** [comment]
I'm afriad it is not clear what this question is asking. Could you rephrase to make it more clear?

**Comment by onesbyones** [comment]
Sorry, it's my mistake. I want to know which code block or java Method is set to the status （KILLED  、SURVIVED 、TIMEOUT... ...）of the mutation test

**Comment by onesbyones** [comment]
> I'm afriad it is not clear what this question is asking. Could you rephrase to make it more clear?

Sorry, it's my mistake. I want to know which code block or java Method is set to the status （KILLED 、SURVIVED 、TIMEOUT... ...）of the mutation test
